xcode/Subconscious/Shared/Library/Prose.swift

```diff
@@ -0,0 +1,66 @@
+//
```

Comment:
General utilities for working with prose. Replaces `truncate.swift`, and adds more utilities.